src/cascadia/TerminalApp/ColorPickupFlyout.cpp

```diff
@@ -0,0 +1,109 @@
+#include "pch.h"
+#include "ColorPickupFlyout.h"
+#include "ColorPickupFlyout.g.cpp"
+#include "winrt/Windows.UI.Xaml.Media.h"
+#include "winrt/Windows.UI.Xaml.Shapes.h"
+#include "winrt/Windows.UI.Xaml.Interop.h"
+#include <LibraryResources.h>
+
+namespace winrt::TerminalApp::implementation
+{
+    // Method Description:
+    // - Default constructor, localizes the buttons and hooks
+    // up the event fired by the custom color picker, so that
+    // the tab color is set on the fly when selecting a non-preset color
+    // Arguments:
+    // - <none>
+    ColorPickupFlyout::ColorPickupFlyout()
+    {
+        InitializeComponent();
+
+        auto okText = RS_(L"Ok");
+        auto customColorText = RS_(L"TabCustomColorChoose");
+        auto clearColorText = RS_(L"TabColorClear");
+
+        btnOk().Content(winrt::box_value(okText));
+        btnCustomColor().Content(winrt::box_value(customColorText));
+        btnClearColor().Content(winrt::box_value(clearColorText));
+
+        customColorPicker().ColorChanged([this](const auto&, const Windows::UI::Xaml::Controls::ColorChangedEventArgs& args) {
```

Comment:
Like my earlier comment, we should probably be doing this like this comment suggests:
https://github.com/microsoft/terminal/issues/3919#issuecomment-564731215

```c++
customColorPicker().ColorChanged({ get_weak(), &ColorPickerFlyout::_ColorChangedHandler });
```

Reply:
Roger